src/main/java/nl/hsac/fitnesse/fixture/util/FtpFileUtil.java

```diff
@@ -17,26 +17,21 @@
 /**
  * Ftp file utilities.
  */
-public class FtpFileUtil {
-    /**
-     * Get size of the FTP file.
-     * @param hostName the FTP server host name to connect
-     * @param port the port to connect
-     * @param userName the user name
-     * @param password the password
-     * @param filePath file to get size for.
-     * @return file's size if found.
-     * @throws RuntimeException in case any exception has been thrown.
+public class FtpFileUtil implements RemoteFileUtil {
```

Comment:
One more thought: I believe apache vfs2 also supports ftp. Would it be hard to rewrite FtpFileUtil to also use vfs2, so that the dependency on commons.net could be removed?

Reply:
I don't know, haven't looked into it. Can it be a todo? I don't know when i will have the time.

Reply:
I don't have access to an ftp server to test whether things work after refactor, so I was hoping you could do this before moving away and forgetting all about it.....

Reply:
I had a look. I'm a bit worried about FtpFileUtil.executeCommandOnFTPServer. Do you want to keep this method or can it be removed?

Reply:
I don't have any tests using it. Does Yarden have any? (I believe the original version of this code also came from you....)

Reply:
Will you have a chance to look at it it further (to see whether commons.net can be removed)? Or ...?

Reply:
Unfortunately i didn't have the time anymore to look at it :(